src/solc_0.8/polygon/child/land/PolygonLandBaseTokenV2.sol

```diff
@@ -608,8 +612,10 @@ abstract contract PolygonLandBaseTokenV2 is IPolygonLand, Initializable, ERC721B
         uint256 x,
         uint256 y
     ) internal {
-        require(x % size == 0 && y % size == 0, "Invalid coordinates");
-        require(x <= GRID_SIZE - size && y <= GRID_SIZE - size, "Out of bounds");
+        require(x % size == 0, "Invalid x coordinate");
+        require(y % size == 0, "Invalid y coordinate");
+        require(x <= GRID_SIZE - size, "Out of bounds");
+        require(y <= GRID_SIZE - size, "Out of bounds");
         if (size == 3 || size == 6 || size == 12 || size == 24) {
             _regroupQuad(from, to, Land({x: x, y: y, size: size}), true, size / 2);
         } else {
```

Comment:
By removing the modifier in various places, and changing the checks where the modifier was used to `_isValidQuad`, you're deleting the following check that was contained in the modifier:
`require(size == 1 || size == 3 || size == 6 || size == 12 || size == 24, "Invalid size");`

1. Is this check deemed unnecessary or should it be reinstated somewhere?
2. Are there tests for the following line in this file `require(false, "Invalid size");`?

Reply:
I believe the check if in _isValidQuad too
https://github.com/thesandboxgame/sandbox-smart-contracts/pull/909/files#diff-71c3c7706fc0944ad90f835d9d45194dbd9fec64c25cd3997d790d35ecad30d3R275